api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -583,6 +583,10 @@ message HttpConnectionManager {
   // *not* the deprecated but similarly named :ref:`stream_error_on_invalid_http_messaging
   // <envoy_v3_api_field_config.core.v3.Http2ProtocolOptions.stream_error_on_invalid_http_messaging>`
   google.protobuf.BoolValue stream_error_on_invalid_http_message = 40;
+
+  // Optional maximum requests for a downstream connection.
+  // If not specified, there is no limit.
```

Comment:
nit: Should this be uint32 instead of google.protobuf.UInt32Value?

We shouldn't expect the default to change away from no-limit.  There is overhead with use of google.protobuf.UInt32Value

Reply:
`ActiveClient` has the implementation to convert zero to unlimited(`std::numeric_limits<uint32_t>::max()`)

Reply:
Changed to uint32